md/cover.md

```diff
@@ -1,11 +1,10 @@
-# Lean 4 Metaprogramming Book
-
-Authors in alphabetical order:
-
-* Arthur Paulino
-* Damiano Testa
-* Edward Ayers
-* Henrik Böving
-* Jannis Limperg
-* Siddhartha Gadgil
-* Siddharth Bhat
+---
+title: "Metaprogramming in Lean 4"
+author: [Arthur Paulino, Damiano Testa, Edward Ayers, Henrik Böving,
+         Jannis Limperg, Siddhartha Gadgil, Siddharth Bhat]
+mainfont: "DejaVu Serif"
+monofont: "DejaVu Sans Mono"
+book: true
+titlepage: true
+keywords: [Lean, theorem proving]
+...
```

Comment:
Please make these changes in the original Lean file (`cover.lean`)

Reply:
Whoops, done.